integration-test/pom.xml

```diff
@@ -3,23 +3,19 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>com.tietoevry.quarkus</groupId>
+        <groupId>io.quarkiverse.resteasy-problem</groupId>
         <artifactId>quarkus-resteasy-problem-parent</artifactId>
-        <version>3.9.1-SNAPSHOT</version>
+        <version>999-SNAPSHOT</version>
```

Comment:
Should this `999` stay here?

Reply:
That's quarkus and quarkiverse convention everywhere.